ophyd/sim.py

```
@@ -139,6 +140,19 @@ def get(self):
         # example.
         return super().get()
 
+    def configure(self,
+                  d: Dict[str, Any]) -> Tuple[Dict[str, Any], Dict[str, Any]]:
+        '''Configure the device for something during a run
+        '''
+        if d:
+            warnings.warn('The `configure` method on `ophyd.sim.SynSignal` '
```

This should probably raise rather than warning.